### PR TITLE
feat(FX-3325): Update the display of size filter labels for saved search M2

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Feature**

This PR resolves [FX-3325]
More context you can find in [Slack](https://artsy.slack.com/archives/C9SATFLUU/p1632216208334400)

#### Example 1
* Values
  * minimum width `5`
  * minimum height `10`
  * maximum height `15`
* Labels
  * `w: from 5 in` or `w: from 5 cm`
  * `h: 10-15 in` or `h: 10-15 cm`

#### Example 2
* Values
  * minimum width `5`
  * maximum width `10`
  * minimum height `15`
  * maximum height `20`
* Label
  * `w: 5-10 in` or `w: 5-10 cm`
  * `h: 15-20 in` or `h: 15-20 cm`

#### Example 3
* Values
  * minimum width `5`
  * minimum height `15`
* Label
  * `w: from 5 in` or `w: from 5 cm`
  * `h: from 15 in` or `h: from 15 cm`

#### Example 4
* Values
  * maximum width `5`
  * maximum height `15`
* Label
  * `w: to 5 in` or `w: to 5 cm`
  * `h: to 15 in` or `h: to 15 cm`

#### iOS
https://user-images.githubusercontent.com/3513494/134199053-8fb78eb6-692e-4136-8875-eea2430a6098.mp4

#### Android
https://user-images.githubusercontent.com/3513494/134201340-07c7c207-aa87-481c-8f4c-da3fa3e1e556.mp4

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Update the display of size filter labels for saved search M2 - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[FX-3325]: https://artsyproduct.atlassian.net/browse/FX-3325

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Update the display of size filter labels for saved search M2 - dimatretyak)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b05e4be214c3d37430261d8cd1fb844b908ef9d1
</p>

File: src/lib/Scenes/SavedSearchAlert/__tests__/helpers-tests.ts [hunk 1]
```diff
@@ -36,6 +36,24 @@ describe("extractPillFromAggregation", () => {
   })
 })
 
+describe("extractSizeLabel", () => {
```
Comment (gkartalis) [inline src/lib/Scenes/SavedSearchAlert/__tests__/helpers-tests.ts:39]: 🤩

**Review by gkartalis — APPROVED** [review]
looks great 🚀

File: src/lib/Scenes/SavedSearchAlert/__tests__/helpers-tests.ts (hunk 1 above)
Comment (gkartalis) [reply src/lib/Scenes/SavedSearchAlert/__tests__/helpers-tests.ts:39]: 💯🍾 thanks for adding tests

**Pull request merged**